src/containers/WelcomePage/components/FrontpageTabsContent.tsx

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (c) 2024-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { TabsContent } from "@ndla/primitives";
+import { styled } from "@ndla/styled-system/jsx";
+
+export const FrontpageTabsContent = styled(TabsContent, {
```

Comment:
Skal denne siden nå hete Frontpage? isåfall kan vi oppdatere navngivning overalt fra WelcomePage

Reply:
Det kan vi godt, men kanskje like greit å gjøre i PR'en der den redesignes?

Reply:
Så ut som du innførte Frontpage her, derfor jeg tenkte det burde gjøres

Reply:
Jepp, men prøver å holde pr'ene mine litt små (som bot for gårsdagens store PR!)